source/Patches/Roles/Modifiers/BigBoiModifier.cs

```diff
@@ -0,0 +1,25 @@
+using TownOfUs.Extensions;
+using TownOfUs.Patches.Roles.Modifiers;
+using UnityEngine;
+
+namespace TownOfUs.Roles.Modifiers
+{
+    public class BigBoiModifier : Modifier, IVisualAlteration
```

Comment:
```suggestion
    public class Giant : Modifier, IVisualAlteration
```
maybe now is the time to change the references to `Giant`?